config.py

```diff
@@ -135,7 +135,7 @@ class TestConfig(Config):
 
     # SQLAlchemy settings
     SQLALCHEMY_DATABASE_URI = TESTING_DB_URL
-    SQLALCHEMY_ECHO = True
+    SQLALCHEMY_ECHO = False
```

Comment:
Should this be based on `.env`? 

I guess it depends on wether you are debugging / analyzing queries right? 

Though in the case of testoutput you can get a lot of interference. It might be nice to be able to turn this on/off without having to change the source code